### PR TITLE
Add vulkan,libgallium_dri,libglapi for mesa 22.0.3

### PR DESCRIPTION
**Pull request by lyintel**
Tracked-On: OAM-101769
Signed-off-by: lyintel <yang.a.lu@intel.com>

### CODE REVIEW

File: graphics/mesa/file_contexts [hunk 1]
```diff
@@ -28,7 +28,9 @@
 /(vendor|system/vendor)/lib(64)?/dri/iris_dri\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/virtio_gpu_dri\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/gallium_dri\.so u:object_r:same_process_hal_file:s0
-/(vendor|system/vendor)/lib(64)?/hw/vulkan\.celadon\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/hw/vulkan\.intel\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/libgallium_dri\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/libglapi\.so u:object_r:same_process_hal_file:s0
```
Comment (renchenglei) [inline graphics/mesa/file_contexts]: Can you help confirm which folder is for libglapi now?
If it is located at /vendor/lib64/dri, then plz help remove line 26

Comment (lyintel) [reply graphics/mesa/file_contexts]: The glapi folder path is : /src/mapi/glapi, so will keep the change

**Review by TianmiChen — APPROVED** [review]

File: graphics/mesa/file_contexts [hunk 2]
```diff
@@ -28,7 +28,9 @@
 /(vendor|system/vendor)/lib(64)?/dri/iris_dri\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/virtio_gpu_dri\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/gallium_dri\.so u:object_r:same_process_hal_file:s0
-/(vendor|system/vendor)/lib(64)?/hw/vulkan\.celadon\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/hw/vulkan\.intel\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/libgallium_dri\.so u:object_r:same_process_hal_file:s0
```
Comment (feijiang1) [inline graphics/mesa/file_contexts:32]: if previous gallium_dri.so is changed to libgallium_dri.so, then need remove gallium_dri.so.

File: graphics/mesa/file_contexts (hunk 1 above)
Comment (lyintel) [reply graphics/mesa/file_contexts]: After double check, libglapi.so still in /vendor/lib64/, so will remove line 33. Thank you chenglei

Comment (lyintel) [reply graphics/mesa/file_contexts]: done.

File: graphics/mesa/file_contexts (hunk 2 above)
Comment (lyintel) [reply graphics/mesa/file_contexts:32]: Can we keep gallium_dri for backward compatibility?

**Review by feijiang1 — APPROVED** [review]

**Review by YuanjunHuang — APPROVED** [review]

**Review by swaroopbalan — APPROVED** [review]

**Pull request merged**